feature_engine/encoding/one_hot.py

```diff
@@ -112,10 +112,10 @@ class OneHotEncoder(BaseCategoricalTransformer):
     """
 
     def __init__(
-        self,
-        top_categories: Optional[int] = None,
-        variables: Union[None, int, str, List[Union[str, int]]] = None,
-        drop_last: bool = False,
+            self,
+            top_categories: Optional[int] = None,
+            variables: Union[None, int, str, List[Union[str, int]]] = None,
+            drop_last: bool = False,
     ) -> None:
```

Comment:
That was not on purpose. I suppose it would have leaked in due to automatic formatting settings per pep-8 in pycharm (my IDE). I can de-indent it. I noticed it after the push and since it passed the flake8 check I left it as is, sorry about that.